packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -376,3 +376,32 @@ vaadin-details {
     }
   }
 }
+
+/**
+ * Improve the appearance of list items.
+ *
+ * @see https://app.clickup.com/t/86b3kq1cc
+ * @since 2025.01.22
+ * 
+ */
+/* stylelint-disable selector-class-pattern */
+.singular-page:not(.slug-blog):not(.woocommerce-page) {
+  .entry-content, .archive-description {
+    ul:not(.gform_fields):not(.glide__slides) {
```

Comment:
Not sure we could deploy this selector to production, it's so complex, nobody can track what happens where.

But indeed, WP content structure across all the different page types is a significant challenge for any broad-stroke selector.

I wonder if there is a sane opt-in mechanism here, some `ul` class like `.cxl-benefits` used to be :thinking:

Reply:
@anoblet suggested using a CSS class for lists but to make editors live easier I decided we'll just make new style default for pages and exclude place where it obviously breaks.

Alternative is to write selector without `:not` but then we need to make lots of overrides to revert styles to defaults in many places

Reply:
I didn't see this before moving the code to WPS.